packages/ui/src/theme/__tests__/defaultTheme.test.ts

```diff
@@ -382,8 +382,8 @@ describe('@aws-amplify/ui', () => {
         --amplify-components-image-height: auto;
         --amplify-components-image-object-fit: initial;
         --amplify-components-image-object-position: initial;
-        --amplify-components-link-active-color: var(--amplify-colors-font-active);
         --amplify-components-link-color: var(--amplify-colors-font-interactive);
+        --amplify-components-link-active-color: var(--amplify-colors-font-active);
```

Comment:
nit: Can we keep this alphabetized?

Reply:
this ordering is based on the design token ordering within the tokens files, we don't seem to have a hard and fast rule yet, but I would prefer to keep those key/value pairs grouped by base tokens and then states
```
{
    color: { value: '{colors.font.interactive.value}' },
   _active: {
    color: { value: '{colors.font.active.value}' },
  },
  _focus: {
    color: { value: '{colors.font.focus.value}' },
  },
  _hover: {
    color: { value: '{colors.font.hover.value}' },
  },
}
```
to alphabetize this list we would either need to rearrange the key/value pairs within the tokens files or sort the output after creating all the variables.  Personally I'd like to just keep it the way it is grouped logically even if that means the results aren't alphabetized, but what does everyone think?